src/Battlescape/ExplosionBState.cpp

```diff
@@ -128,7 +132,7 @@ void ExplosionBState::init()
 				_power = 0;
 				miss = true;
 			}
-			else
+			else if (type != BT_SCRIPTED)
```

Comment:
why is this here? no message to the user on panic/mc?

Reply:
The reason was, that I was using the scripted item as a method to have friendly/enemy remote targeting. So I wanted to have the targeting features of panic/mc but not the effect itself.